composer.json

```
@@ -20,5 +20,10 @@
     "scripts": {
         "fix-cs": "php-cs-fixer fix --config=.php-cs-fixer.php -v --show-progress=dots",
         "check-cs": "@fix-cs --dry-run"
+    },
+    "config": {
+        "allow-plugins": {
+            "php-http/discovery": true
+        }
```

We try to explicitly limit plugins executed during composer installation, so we disable them by default.
```suggestion
        "allow-plugins": false
```